wordle-ad-farm/README.md

```diff
@@ -1,6 +1,7 @@
 # wordle-ad-farm
 ## Azure Deployment
 You can view the deployed project [here](https://polite-desert-067c8041e.1.azurestaticapps.net).
+https://polite-desert-067c8041e.1.azurestaticapps.net/
```

Comment:
Would have been nice to see this as a comment as well. Love the random name polite-desert!